index.html

```diff
@@ -4,27 +4,187 @@
+                    </ul>
+                </div>
+            </div>
+            <main>
+                <div id="work-heading">
+                    <label>My Recent Works</label>
+                </div>
+                <div class="works">
+                    <div class="work snapshot">
+                        <img src="./images/work1.jpg" alt="My First Project">
+                    </div>
+                    <div class="work project-name">
+                        <h2>Multi Post Stories</h2>
+                    </div>
+                    <ul class="work project-summary">
+                        <li>Canopy</li>
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                    <div class="work description">
+                        <p>
+                            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+                        </p>
+                    </div>
+                    <div class="work languages">
+                        <label>CSS</label>
+                        <label>HTML</label>
+                        <label>Bootstrap</label>
+                        <label>Ruby</label>
+                    </div>
+                    <a class="work see-project" href="">See Project</a>
+                </div>
+                <div class="works">
+                    <div class="work snapshot">
+                        <img src="./images/Illustration.svg" alt="My First Project">
+                    </div>
+                    <div class="work project-name">
+                        <h2>Personal Art Printing Data</h2>
+                    </div>                
+                    <ul class="work project-summary">
+                        <li>Canopy</li>
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>                
+                    <div class="work description">
+                        <p>
+                            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+                        </p>
+                    </div>
+                    <div class="work languages">
+                        <label>CSS</label>
+                        <label>HTML</label>
+                        <label>Bootstrap</label>
+                        <label>Ruby</label>
+                    </div>
+                    <a class="work see-project" href="">See Project</a>
+                </div>
+                <div class="works">
+                    <div class="work snapshot">
+                        <img src="./images/schedule.jpg" alt="My First Project">
```

Comment:
- [ ] Kindly Also note that its a requirement that you use the colors, images from the figma design as much as possible. Your colors  and images are currently differing